src/terraform/linode/terraform.tf

```
@@ -0,0 +1,11 @@
+terraform {
```

SHOULD: We should declare `terraform_version`.
In my experience, I have broken tfstate when upgrading or applying multiple people.

```suggestion
terraform {
    required_version = "~> 1.11.0"
```

---

WANT: If developers are using [tfenv](https://github.com/tfutils/tfenv), it is helpful to have `.terraform-version` file.

💾./src/terraform/.terraform-version or ./src/terraform/linode/.terraform-version
```
latest-allowed
```

---

@nakakoh 
could you tell me how the tfstate was broken for me to understand this fix?
was the team working without backend?

---

also I'm not sure if it's good to have a tool-specific file here like `.terraform-version` which is only valid file for tfenv users eventhough nothing related with tfenv is in this repository

---

@hirokazutomoda 
> could you tell me how the tfstate was broken for me to understand this fix?
was the team working without backend?

In my experience, tfstate often breaks when updating terraform. Even if the team don't use backend.
At least I would like to add `required_version` to `terraform.tf`.

> also I'm not sure if it's good to have a tool-specific file here like .terraform-version which is only valid file for tfenv users eventhough nothing related with tfenv is in this repository

Certainly if the team doesn't use tfenv, they don't need `.terraform-version`. But when the team uses multiple versions of terraform for other projects, it is useful to use tfenv.
So it may be good to add the tfenv install processes to the README.
Let's discuss this matter in the other issue or PR. 🙇‍♂️

---

required_version added!